test/jdk/java/awt/Window/Grab/GrabTest.java

```diff
@@ -56,9 +71,13 @@ public class GrabTest {
 
     static volatile boolean passed = true;
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
+
+        robot = new Robot();
+        robot.setAutoDelay(500);
```

Comment:
@alisenchung The stability test fix works on macOS 14 both with setAutoDelay(500) & setAutoDelay(100).
Any reason why the autodelay is set to high number here since your earlier one-off delay is 100ms?

Reply:
Updated commit changes not seen on the PR.

Reply:
I don't see the 100 ms changes either, looks like you didn't push the commit.

Reply:
oops! just pushed